client/src/test/java/io/getstream/chat/android/client/api/ClientConnectionTests.kt

```diff
@@ -77,6 +79,7 @@ internal class ClientConnectionTests {
         logger = mock(ChatLogger::class.java)
         notificationsManager = mock(ChatNotifications::class.java)
         initConnectionListener = mock(InitConnectionListener::class.java)
+        attachmentHelper = mock(AttachmentHelper::class.java)
```

Comment:
Let's update these `mock` calls to use the reified function instead of using a `Class` parameter too, since we've done it in `MockClientBuilder` too 🙂

Reply:
Done